docs/pages/admin-guides/access-controls/access-requests/automatic-reviews.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Configure Automatic Reviews
+description: Describes how to configure Access Monitoring Rules for Automatic Reviews.
+---
+
+Teleport supports automatic reviews of Access Requests for Teleport roles. This
+feature enables teams to enforce a zero standing privilege policy, while
+still allowing users to receive temporary access without manual approval.
+
+## How it works
+
+Automatic reviews are triggered by Access Monitoring Rules. These rules instruct
+Teleport to monitor specific Access Requests and automatically submit a review
+when certain conditions are met. The conditions can be configured based on the
+requested roles, and based on the requesting user's Teleport traits.
+
+For example, an Access Monitoring Rule can configure automatic Access Request
+approval when a user with the Teleport traits `team: demo` requests access to
+the `access` role.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- Teleport Identity Governance enabled on your account.
```

Comment:
It's not clear that "account" here refers to your Teleport subscription and not your user account. What if we just said:
```suggestion
- This feature requires Teleport Identity Governance.
```